core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -179,18 +181,21 @@ class RenderBlockingResources extends Audit {
     }
 
     if (!results.length) {
-      return {results, wastedMs: 0};
+      return {results, fcpWastedMs: 0, lcpWastedMs: 0};
     }
 
-    const wastedMs = RenderBlockingResources.estimateSavingsWithGraphs(
+    const fcpWastedMs = RenderBlockingResources.estimateSavingsWithGraphs(
       simulator,
       fcpSimulation.optimisticGraph,
       deferredNodeIds,
       wastedCssBytes,
       artifacts.Stacks
     );
 
-    return {results, wastedMs};
+    const lcpRecord = await LCPImageRecord.request(metricComputationData, context);
+
+    // In most cases if the LCP is an image, render blocking resources don't affect LCP. For these cases we should reduce its impact.
+    return {results, fcpWastedMs, lcpWastedMs: lcpRecord ? 0 : fcpWastedMs};
```

Comment:
Can you add a test case that covers the text case? Should be able to use [this](https://github.com/GoogleChrome/lighthouse/blob/8b119b143413bc3d3ea0384ec2f180f5a1f9f7a2/core/test/computed/metrics/lcp-breakdown-test.js#L13C1-L13C1).